presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestTransformExistsApplyToScalarApply.java

```diff
@@ -40,23 +40,24 @@
 
 public class TestTransformExistsApplyToScalarApply
 {
-    private final TypeRegistry typeManager = new TypeRegistry();
-    private final FunctionRegistry registry = new FunctionRegistry(typeManager, new BlockEncodingManager(typeManager), new FeaturesConfig());
-    private final Rule transformExistsApplyToScalarApply = new TransformExistsApplyToScalarApply(registry);
-
     private RuleTester tester;
+    private Rule transformExistsApplyToScalarApply;
 
     @BeforeClass
     public void setUp()
     {
         tester = new RuleTester();
+        TypeRegistry typeManager = new TypeRegistry();
+        FunctionRegistry registry = new FunctionRegistry(typeManager, new BlockEncodingManager(typeManager), new FeaturesConfig());
+        transformExistsApplyToScalarApply = new TransformExistsApplyToScalarApply(registry);
```

Comment:
Inline this in test body, to mimic behavior of other tests. Then, you don't need to `null` it in `tearDown` method.